src/routes/resources/teams/+page.svelte

```diff
@@ -3,37 +3,36 @@
 		>Microsoft Teams</Heading
 	>
 	<Hr />
-	<p class="pb-4">
-		Wayne State College ACM communicates and stores files using Microsoft Teams. Once a member of
-		ACM you will have access to use the ACM Team in Teams. Teams allows us to branch out into
-		different channels so we have a conversations for said channel. Everyone has access to all of
-		the main channels except for the ACM Officers and ACM Officers Tasks channels as those are only
-		for ACM Officers. Channels to keep an eye on are <b
-			>General, Internship Opportunities, and any of the subcommittee channels</b
-		>.
-	</p>
-	<p class="pb-4">
-		If you are joining ACM it is recommended that you download Teams on your computer or phone as a
-		majority of special updates and messages will presented there. Teams also allows you to message
-		people individually or in groups, which helps cut down the amount of emails being sent back and
-		forth.
-	</p>
-	<p class="pb-4">
-		All Wayne State College Students have access to Teams through their myWSC. To view Teams on your
-		browser all you have to do is log into your <b>myWSC</b>, go to
-		<a href="https://www.wsc.edu/current-students">Resources for Current Students</a>, click on
-		<b>Show More Applications</b>, then scroll down until you find <b>Mircosoft Teams</b>. To
-		download Microsoft Teams click
-		<a href="https://www.microsoft.com/en-us/microsoft-teams/log-in">here</a>. Make sure you sign in
-		with your Wayne State College account. For more information about Microsoft Teams click
-		<a href="https://www.wsc.edu/directory/31/help_sheets/category/957/categoryInfo/35">here</a>.
-	</p>
-</div>
+	
+	<div class="pb-4">
+		If you are joining <b>ACM</b> it is recommended that you download <b>Teams</b> on your computer or phone as a majority of special updates and messages will presented there. All <b>Wayne State College</b> students have access to <b>Teams</b> through their <b>myWSC</b>. To have access to the <b>ACM Team</b> talk to or message an <b>ACM officer</b> to add you.
+	</div>	
+	<div class="flex flex-wrap">
+		<div>
+			<b>To view Teams on browser:</b>
+			<p class="pb-8">
+				1. Log into your <b>myWSC</b> account <br>
+				2. Go to <a href="https://www.wsc.edu/current-students" target="blank">Resources for Current Students</a> <br>
+				3. Click on <b>Show More Applications</b> <br>
+				4. Scroll down until you find <b>Microsoft Teams</b>
+			</p>
+			<b>To download Microsoft Teams</b>
+			<p>
+				1. Go to the <a href="https://www.microsoft.com/en-us/microsoft-teams/download-app" target="blank">Microsoft Teams download page</a> <br>
+				2. Click Download for desktop (or Download for mobile if needed) <br>
+				3. Open the downloaded file and follow the installation prompts. <br>
+				4. Launch <b>Microsoft Teams</b> and sign in with your <b>Wayne State College</b> account
+			</p>
+		</div>
+		<div>
 
+		</div>
+	</div>
+	</div>
 <style>
```

Comment:
The default text color for links is fine but in the future if you want to change text color look at tailwind